packages/cli/src/commands/build.ts

```diff
@@ -5,10 +5,21 @@ import { existsSync } from 'fs'
 import { copySync, moveSync, readdirSync, removeSync } from 'fs-extra'
 import { withBasePath } from '../utils/directory'
 import { generate } from '../utils/generate'
+import { ArgInput } from '@oclif/core/lib/interfaces'
```

Comment:
question: why did you add this to this command, but not to the others?

Reply:
By mistake! I've taken it out!